fastlane_core/lib/fastlane_core/configuration/config_item.rb

```diff
@@ -138,6 +149,7 @@ def initialize(key: nil,
 
       @key = key
       @env_name = env_name
+      @env_names = [env_name].compact + (env_names || [])
```

Comment:
maybe use `([env_name] + (env_names || [])).compact` in case `env_names` contain `nil`.

`([env_name] + env_names).compact` would also work by using the side effect of `compact`. Maybe too smart.